packages/sample/src/main.ts

```diff
@@ -18,6 +20,18 @@ if(options.list) {
     });
 }
 
+if(options.motor) {
+    let power: number = Number(options.motor);
+
+    const hubs: ExpansionHub[] = await getConnectedExpansionHubs();
+    let hub = hubs[0]
+
+    await hub.setMotorChannelMode(1, 0, true);
+    console.log(`Power is ${power}, type is ${typeof power}`)
+    await hub.setMotorConstantPower(1, power);
+    await hub.setMotorChannelEnable(1, true);
```

Comment:
Methods used in the sample must be documented. Also, they should accept values from -1 to 1, not whatever it is now.

Reply:
Updated it to [-1,1]

Reply:
Added docs